server/routerlicious/packages/services-telemetry/src/sanitizationLumberFormatter.ts

```diff
@@ -9,7 +9,43 @@ import { Lumber } from "./lumber";
 /**
  * @internal
  */
-export class SanitizationLumberFormatter implements ILumberFormatter {
+export class BaseSanitizationLumberFormatter implements ILumberFormatter {
+	readonly redactedStr = "[LUMBER_REDACTED]";
+
+	public transform(lumber: Lumber<string>): void {
+		if (lumber.exception) {
+			const sensitiveKeys = this.sanitizeCommonCases(lumber.exception);
+
+			if (sensitiveKeys) {
+				lumber.setProperty("detectedSensitiveKeys", sensitiveKeys);
```

Comment:
we are also adding detectedSensitiveKeys at line 70. Do we need to differentiate between the two?

Reply:
I was thinking about that as well. Given that the Base class is for known cases and we are interested on keep adding cases when we find new ones, I think it is best to only add detectedSensitiveKeys in the subclass, which searches for unknown scenarios were data is leaked. Changed.